lib/mongo/error/notable.rb

```diff
@@ -70,6 +70,13 @@ def add_note(note)
       # @api experimental
       attr_accessor :service_id
 
+      # Returns global id of the connection on which the error occurred.
+      #
+      # @return [ Object | nil ] Service id.
+      #
+      # @api experimental
```

Comment:
Could be private if you don't have a use for applications to read it.

Reply:
Done